Task3_Products/routers/categories.py

```diff
@@ -0,0 +1,73 @@
+from typing import Literal, Optional
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from Task3_Products import crud, models, schemas
```

Comment:
By this way you import a lot of unnecessary objects, let's import only needed ones